dashboard/src/components/TreeListingPage/TreeListingPage.tsx

```diff
@@ -38,7 +38,11 @@ const TreeListingPage = ({ inputFilter }: ITreeListingPage): JSX.Element => {
       return [];
     } else {
       return data
-        .filter(tree => tree.git_commit_hash?.includes(inputFilter))
+        .filter(
+          tree =>
+            tree.git_commit_hash?.includes(inputFilter) ||
+            tree.tree_names.includes(inputFilter),
```

Comment:
partial searching by tree name is not working because tree_names is a array here